src/Symfony/Component/JsonEncoder/ValueTransformer/ValueTransformerInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\ValueTransformer;
+
+use Symfony\Component\TypeInfo\Type;
+
+/**
+ * Transforms data between PHP and JSON and to other way around during the encoding/decoding process.
```

Comment:
It's not clear to me what a "json value" is. Can we be more explicit? It's a PHP value that can be json encoded? e.g. can objects we returned?

Reply:
Indeed, a "JSON value" is a value that is ready to be JSON encoded (so even objects can be returned if needed).
I updated the description accordingly, thanks